ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -18,6 +18,13 @@ interface {{ ethernet_interface }}
 {%             if ethernet_interfaces[ethernet_interface].mtu is defined and ethernet_interfaces[ethernet_interface].mtu != 1500 %}
    mtu {{ ethernet_interfaces[ethernet_interface].mtu }}
 {%             endif %}
+{%             if ethernet_interfaces[ethernet_interface].logging_event is defined and ethernet_interfaces[ethernet_interface].logging_event is not none %}
```

Comment:
This feature has been added for v2 with a different approach (https://github.com/aristanetworks/ansible-avd/pull/570)

```yaml
  Ethernet6:
    logging:
      event:
        link_status: true
```

Reply:
Tks. I had missed this PR #570. I changed it accordingly.